betas.go

```diff
@@ -0,0 +1,99 @@
+package linodego
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/linode/linodego/internal/parseabletime"
+)
+
+// Beta Program is a new product or service that is not generally available to all Akamai customers.
+// Users must enroll into a beta in order to access the functionality.
+type BetaProgram struct {
+	Label       string `json:"label"`
+	ID          string `json:"id"`
+	Description string `json:"description"`
+
+	// Start date of the beta program.
+	StartDT *time.Time `json:"-"`
+
+	// End date of the beta program.
+	EndDT *time.Time `json:"-"`
+
+	// Greenlight is a program that allows customers to gain access to
+	// certain beta programs and to collect direct feedback from those customers.
+	GreenlightOnly bool `json:"greenlight_only"`
+
+	// Link to product marketing page for the beta program.
+	MoreInfo string `json:"more_info"`
+}
+
+// BetasPagedResponse represents a paginated Beta Programs API response
+type BetasPagedResponse struct {
+	*PageOptions
+	Data []BetaProgram `json:"data"`
+}
+
+// endpoint gets the endpoint URL for BetaProgram
+func (BetasPagedResponse) endpoint(_ ...any) string {
+	return "/betas"
+}
+
+// UnmarshalJSON implements the json.Unmarshaler interface
+func (beta *BetaProgram) UnmarshalJSON(b []byte) error {
+	type Mask BetaProgram
+
+	p := struct {
+		*Mask
+		StartDT *parseabletime.ParseableTime `json:"start_dt"`
+		EndDT   *parseabletime.ParseableTime `json:"end_dt"`
```

Comment:
```suggestion
		Started *parseabletime.ParseableTime `json:"started"`
		Ended   *parseabletime.ParseableTime `json:"ended"`
```

Reply:
Makes sense. I used StartDT and EndDT is because the sample response in API spec doc use them. I'll double check with the response from alpha.